lib/v2/nber/radar.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    'nber.org': {
+        __name: 'National Bureau of Economic Research',
+        '.': [
+            {
+                title: 'New working paper',
+                docs: 'https://docs.rsshub.app/en/journal.html#national-bureau-of-economic-research',
+                source: ['/news'],
```

Comment:
`https://www.nber.org/news` does not exist
```suggestion
                source: ['/papers'],
```